docker/go-control-plane/testcontrolplane/callbacks.go

```diff
@@ -0,0 +1,145 @@
+package testcontrolplane
+
+import (
+	"context"
+	"log"
+	"os"
+	"sync"
+
+	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
+	"github.com/envoyproxy/go-control-plane/pkg/server/v3"
+)
+
+type Callbacks struct {
```

Comment:
There is absolutely no documentation for any of the types in this package. How am I supposed to understand what is happening here?